internal/annotation/load_balancer.go

```diff
@@ -186,6 +186,10 @@ const (
 	// performing the health check.
 	LBSvcHealthCheckHTTPPath Name = "load-balancer.hetzner.cloud/health-check-http-path"
 
+	// LBSvcHealthCheckTLSPort specifies the port for which to enable TLS
+	// performing the health check.
+	LBSvcHealthCheckTLSPort Name = "load-balancer.hetzner.cloud/health-check-tls-port"
```

Comment:
What do you think about naming this similarly to `LBSvcHealthCheckHTTPValidateCertificate`/`load-balancer.hetzner.cloud/health-check-http-validate-certificate`? These options basically do the same, just that the new option is restricting to a single port.